### ISSUE TITLE
Questions about DTWNet

### ISSUE THREAD

**Issue by yjhong89** [TideDancer/DTWNet#2]
Hi.

I am YJHong.

I found your nice work while I am trying to solve time sequence matching problem (just for matching, no need to classify sequence).

I have few questions about DTWNet.

1. When I see code, especially regarding UCR dataset, it seems like all training data have "equal" length while my dataset have difference lengths for each sequence. Can you give me a guide in this case?

2. Does code work only in torch version under 1.0.0 (0.4.x)? I am trying to run build_dp.py in '_ext/script/', there is ImportError stating torch.utils.ffi is deprecated. 

Thank you in advance.

Regards.

**Comment by TideDancer** [comment]
Hi YJHong, thank you for your interest. 
For the UCR dataset, I found some of them have variable length, e.g. AllGestureWiimoteX,Y,Z. But most of them have fixed length. For the variable length case I think padding (pytorch pad_sequence() function could be useful) is a good option. 
For the c-ffi, it seems indeed the latest pytorch version depreciated c-ffi. One option is to use the new cpp_extension (torch.utils.cpp_extension) to compile.
Hope these helps.
